app/config/base.json

```diff
@@ -18,5 +18,25 @@
 	  }
 	},
 	"specialAvatars": {
+	},
+	"map": {
+		"stats": {
+			"numberInHouse": {
+				"display": true,
+				"text": "{0} in house",
+				"emoji": "🏚️"
+			},
+			"numberInRooms": {
+				"display": true,
+				"text": "{0} in rooms",
+				"emoji": "🚪"
+			},
+			"numberInHallways": {
+				"display": true,
+				"text": "{0} roaming",
+				"emoji": "🌓"
```

Comment:
We could maybe just combine the text and emoji keys:

```json
{
   "emoji": "🚪  {0} in rooms"
}
```

Reply:
i am in favor of this